### ISSUE TITLE
Add a birth control pill reminder feature

### ISSUE THREAD

**Issue by friesenkiwi** [arnowelzel/periodical#26]
Would it be interesting/desirable for you, to add a configurable option to have a reminder each day during the non-menstruation period on a specific time of day? ("Pillenwecker")
This would be very helpful for people to remind them to take their birth control pill regularly. A standard reminder or alarm clock is not very helpful, because it reminds also during menstruation and will not sustain the reminder, so one can decide "I'll take it in ten minutes".
If it's interesting for this app, I'll volunteer to implement.

**Comment by arnowelzel** [comment]
Yes, this might be an interesting feature. You're welcome to contribute code - thank you!

**Comment by Hilary325** [comment]
This is a good idea. I'm sure, a reminder can be very helpful. When I took birth control pills, sometimes I forgot about them.  As I wasn't ready for the baby at that time, I started using Implant Birth Control.  If I had such a reminder, maybe I didn't switch.
